ang/civicase/ActivitiesCalendar.js

```diff
@@ -580,6 +610,20 @@
       load({ useCache: false });
     }
 
+    /**
+     * Creates the url for the "see more" link, based on the given date
+     * (the link will send the user to the activity feed, filtered by that date)
+     *
+     * @return {Date} date
+     * @return {TrustedValueHolderType}
+     */
+    function seeMoreLinkUrl (date) {
+      var urlParams = getSeeMoreQueryParams(date);
+      urlParams.af = JSON.stringify(urlParams.af);
```

Comment:
this is something that should be returned by the `getSeeMoreQueryParams`. The `seeMoreLinkUrl` should not be aware of any query params so it's kept simple.